shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -110,6 +111,7 @@
     private FlutterNativeView mNativeView;
     private final AnimationScaleObserver mAnimationScaleObserver;
     private boolean mIsSoftwareRenderingEnabled = false; // using the software renderer or not
+    private final LongSparseArray<PointerState> mPointers;
```

Comment:
Why LongSparseArray instead of a plain HashMap?

Reply:
android lint suggested it.

Reply:
Ah, cool. Just read up about it. Sounds like it's the better choice. Let's keep it.